src/components/App.js

```diff
@@ -1,35 +1,42 @@
 import React from 'react';
+import ReactDOM from "react-dom";
 import {Switch, Route} from 'react-router-dom';
 
 import Form from 'components/Form';
 import { Layout, LayoutRow } from 'components/Layout';
 import ManageAppointment from 'components/appointments/ManageAppointment';
 
+import LanguageSelection from "components/LanguageSelection";
 
 /*
 Top level router - routing between an actual form or supporting screens.
  */
-const App = (props) => {
+const App = ({ languageSelectorTarget, ...formProps }) => {
```

Comment:
```suggestion
const App = ({ languageSelectorTarget, ...props }) => {
```
will also still work - don't think this rename is particularly needed